csrc/selective_scan/selective_scan_common.h

```diff
@@ -4,10 +4,40 @@
 
 #pragma once
 
-#include <cuda_bf16.h>
+#ifndef USE_ROCM
+    #include <cuda_bf16.h>
+#else
+    #include <hip/hip_bf16.h>
+#endif
```

Comment:
We really should inform the pytorch team (me, et al) to add this to our cuda to hip mappings file so you don't have to do the hipify here in the sources, but let pytorch hipify do it.

Reply:
Yes, this mapping was already pushed to "cuda_to_hip_mappings" there some time ago and should work with the latest "pytorch main" built from source. We left this ifndef is there for backward compatibility. We were not 100% sure how pytorch releases work in that aspect, i.e. starting with which pytorch version will the users automatically get the desired behavior.

Reply:
I'm okay leaving this then for backward-compat.  Thanks for the explanation.  Makes sense.